onboarding-workflow_v5/bin/upload_repo.sh

```diff
@@ -76,4 +96,9 @@ if  diff --brief --recursive "repo" "repo_clone" --exclude ".git" --no-dereferen
 fi
 log "Compare status: $DIFF_STATUS"
 
+if (( $DIFF_STATUS == 0 )); then
```

Comment:
this command seems to be called inside the path we want to remove, does it work?